#! /bin/sh /usr/share/dpatch/dpatch-run
## 14-l2tpd-keepalive.dpatch, by Mikko Varso <mikko.varso@codebay.fi>
##
## All lines beginning with `## DP:' are a description of the patch.

@DPATCH@

--- l2tpd-0.69.orig/l2tp.h
+++ l2tpd-0.69/l2tp.h
@@ -44,7 +44,7 @@
 #define DEF_MAX_TUNNELS 32      /* By default only allow this many
                                    tunnels to exist */
 
-#define HELLO_DELAY 60          /* How often to send a Hello message */
+#define HELLO_DELAY (0x007FFFFF)/* How often to send a Hello message */
 
 struct control_hdr
 {
